### PR TITLE
Missing documentation for enabling HTTPS connector

### PR DESCRIPTION
**Pull request by rpelisse**
Includes somes fixes and changes to add validation of connectors paramters.

### CODE REVIEW

**Review by guidograzioli — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -144,6 +144,28 @@ All that you have to do to enable a mod_cluster listener for jws is to edit the
 - `tomcat_modcluster_ip` (Set the ip of the mod_cluster instance)
 - `tomcat_modcluster_port` (Set the port of the mod_cluster instance)
 
+(This feature is validated and tested by the following [Molecule scenario](https://github.com/ansible-middleware/jws-ansible-playbook/tree/main/molecule/ajp) )
+
+## Enable HTTPS
+
+The default template for server.xml provided with this Ansible collection already includes the required configuration to use HTTPS. It just need to be activated. However, the collection does not build, nor provide the requires SSH keys and Java Keystore. It expects it to be already installed and available.
```
Comment (csutherl) [inline README.md:151]: There's a typo here, "provide the requires" should be "provide the required".

File: README.md [hunk 2]
```diff
@@ -144,6 +144,28 @@ All that you have to do to enable a mod_cluster listener for jws is to edit the
...
+
+    tomcat_listen_https_enabled: True
+    #tomcat_listen_https_keystore_file: /etc/ssl/keystore.jks
```
Comment (csutherl) [inline README.md:154]: Typo here, remove the proceeding '#' so that the line isn't commented.

File: README.md [hunk 3]
```diff
@@ -144,6 +144,28 @@ All that you have to do to enable a mod_cluster listener for jws is to edit the
...
+
+Please refers to the [server documentation](https://tomcat.apache.org/tomcat-9.0-doc/ssl-howto.html#Quick_Start) for more details on the setup and configuration of this feature.
+
+Note: There other collections and modules available to automate the creation of those files (such as [Ansible OpenSSH Keypair collection](https://docs.ansible.com/ansible/latest/collections/community/crypto/openssh_keypair_module.html), [Ansible Collection Community Crytpo](https://docs.ansible.com/ansible/latest/collections/community/crypto/index.html) and the [Java Keystore module](https://docs.ansible.com/ansible/latest/collections/community/general/java_keystore_module.html)). Please refers to those in order to automate this part.
+
+(This feature is validated and tested by the following [Molecule scenario](https://github.com/ansible-middleware/jws-ansible-playbook/tree/main/molecule/https) )
+
+## Overridng the default template for server.xml
```
Comment (csutherl) [inline README.md:162]: Typo, "Overridng" should be "Overriding".

File: README.md (hunk 2 above)
Comment (rpelisse) [reply README.md:154]: @csutherl this was NOT a typo, it was meant to indicate the default value is this one (like it's done in many configuration file, like the ones for HTTPd). But maybe you are right, it's not that clear...

**Pull request merged**